docs/source/models/gbd2017/causes/neonatal_other/index.rst

```diff
@@ -36,3 +36,38 @@ neonatal disorders.
 
 .. todo::
    Describe cause model
+
+Restrictions
+------------
+
+.. list-table:: Restrictions
+	:widths: 15 15 20
+	:header-rows: 1
+
+	* - Restriction type
+	  - Value
+	  - Notes
+	* - Male only
+	  - False
+	  -
+	* - Female only
+	  - False
+	  -
+	* - YLL only
+	  - False
+	  -
+	* - YLD only
+	  - False
+	  -
+	* - YLL age group start
+	  - Early neonatal
+	  - age_group_id = 2; [0-0.01917808 years)
+	* - YLL age group end
+	  - Post neonatal
+	  - age_group_id = 4; [0.07671233-1 years)
+	* - YLD age group start
+	  - Early neonatal
+	  - age_group_id = 2; [0-0.01917808 years)
+	* - YLD age group end
+	  - 90-94
+	  - age_group_id = 32; [90-94 years)
```

Comment:
Should be [90-95 years)

Reply:
I thought it is 95 plus.

Reply:
Good catch, @XiaoluQianUW , looks like it should be age group id 235, not 32

Reply:
90-94 is right.  It's a demographer's notation thing. It doesn't really mix with the math open/closed interval notation.

Reply:
@collijk There are two issues here:
1. GBD mapping says the age_group_id for YLD age group end for other neonatal disorders should be 235, not 32.
2. If the correct age group id _were_ 32, the table already includes the correct age group name, "90-94" in the "Value" column. The interval appears in the "Notes" column, presumably as a clarification to the human reader that this age group actually includes all ages up to but not including 95.